pkg/api/plugin.go

```diff
@@ -23,6 +23,7 @@ const (
 	PluginStepInitialize                          PluginStep = "Initialize"
 	PluginStepInitializeUpdateBlob                PluginStep = "InitializeUpdateBlob"
 	PluginStepResetUpdateBlob                     PluginStep = "ResetUpdateBlob"
+	PluginStepBlobDoesNotExist                    PluginStep = "BlobDoesNotExist"
```

Comment:
maybe this would be better 
`PluginStepCheckEtcdBlobExists                    PluginStep = "CheckEtcdBlobExists"`